core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt

```diff
@@ -103,14 +102,14 @@ class DrillAdminEndpoints(override val kodein: Kodein) : KodeinAware {
             authenticate {
                 val meta = "Configure agent logging levels"
                     .examples(
-                        example("Agent logging configuration", defaultLoggingConfig)
+                        example("Agent logging configuration", LoggingConfigDto(level = LogLevel.INFO))
                     )
                     .responds(
                         ok<Unit>(), notFound(), badRequest()
                     )
-                put<ApiRoot.Agents.AgentLogging, LoggingConfig>(meta) { (_, agentId), loggingConfig ->
+                put<ApiRoot.Agents.AgentLogging, LoggingConfigDto>(meta) { (_, agentId), (logLevel) ->
                     logger.debug { "Attempt to configure logging levels for agent with id $agentId" }
-                    loggingHandler.updateConfig(agentId, loggingConfig)
+                    loggingHandler.updateConfig(agentId, logLevel.toConfig())
```

Comment:
```suggestion
                    loggingHandler.updateConfig(agentId, loggingConfigDto)
```

change the handler, so it will support loggingConfigDto